lib/models/addon.js

```diff
@@ -41,6 +42,13 @@ heimdall.registerMonitor('addon-tree-cache', function AddonTreeCacheSchema() {
   this.adds = 0;
 });
 
+heimdall.registerMonitor('cache-key-for-tree', function CacheKeyForTreeSchema() {
+  this.optOut = {
+    modifiedMethods: 0,
+    treeForMethodsOverride: 0,
+  };
```

Comment:
Can we change this schema to just have simple numeric properties (counts) rather than a pojo?

Aside from anything else it will make upgrading to v0.3 heimdall more straightforward